qutebrowser/completion/completiondelegate.py

```diff
@@ -198,8 +198,8 @@ def _get_textdoc(self, index):
             columns_to_filter = index.model().srcmodel.columns_to_filter
             if index.column() in columns_to_filter and pattern:
                 repl = r'<span class="highlight">\g<0></span>'
-                text = re.sub(re.escape(pattern), repl, self._opt.text,
-                              flags=re.IGNORECASE)
+                text = re.sub(re.escape(pattern).replace(r'\ ', r'|'),
+                              repl, self._opt.text, flags=re.IGNORECASE)
```

Comment:
@epitron I had that originally, but I didn't like that it is "greedy" so anything between terms is highlighted. It can be better for user though.